addons/web/static/src/scss/ui.scss

```diff
@@ -74,12 +74,28 @@
     animation: catchAttention 200ms ease 0s infinite normal;
 }
 
+// bounce effect
 @keyframes catchAttention {
+    0%, 20%, 40%, 60%, 80%, 100% {
+        transition-timing-function: cubic-bezier(0.215, 0.610, 0.355, 1.000);
+    }
```

Comment:
I stumbled upon this. Just by curiosity:
- What is the point of *animating* `transition-timing-function`? and to the same value at different points all the way? 
- Has this any effect at all since it is an animation and not a transition?

Reply:
@aab-odoo Still want to annoy you with this :smiling_imp:  :point_up:

Reply:
No idea. Maybe @msh-odoo or @jip-odoo would know?